team-telnyx/webrtc
# feat(WEBRTC): improve react-native layout

Added navigation 
Changed layout to brand Telnyx

## 📝 To Do


## ✋ Manual testing

1. cd examples/react-native
2. Connect your real device
3. Run npm run android.
4. See if it is with logos and colors about Telnyx.
5. Get your production credentials and try to make a call

## 🦊 Browser testing

### Desktop
